src/pages/workspace/accounting/netsuite/import/NetSuiteImportPage.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import {View} from 'react-native';
+import ConnectionLayout from '@components/ConnectionLayout';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import OfflineWithFeedback from '@components/OfflineWithFeedback';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {updateNetSuiteSyncTaxConfiguration} from '@libs/actions/connections/NetSuiteCommands';
+import * as ErrorUtils from '@libs/ErrorUtils';
+import {isSyncTaxEnabled} from '@libs/PolicyUtils';
+import withPolicyConnections from '@pages/workspace/withPolicyConnections';
+import type {WithPolicyConnectionsProps} from '@pages/workspace/withPolicyConnections';
+import ToggleSettingOptionRow from '@pages/workspace/workflows/ToggleSettingsOptionRow';
+import * as Policy from '@userActions/Policy/Policy';
+import CONST from '@src/CONST';
+
+function NetSuiteImportPage({policy}: WithPolicyConnectionsProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const policyID = policy?.id ?? '-1';
+    const config = policy?.connections?.netsuite?.options.config;
+    const importFields = CONST.NETSUITE_CONFIG.IMPORT_FIELDS;
```

Comment:
NAB: Can we just use the constant directly in the code? Especially because I see we use this just once.

Reply:
done.